contents/docs/plugins/build/overview.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Overview
+sidebar: Docs
+showTitle: true
+---
+
+A great way to understand what you can do with plugins is to understand how data flows throw plugins.
```

Comment:
Maybe worth adding a link to the page explaining what plugins are if we e.g. send this page to someone instead of them navigating here (in which case they probably read about plugins before)